src/models/launch_1/customization.md

```diff
@@ -0,0 +1,42 @@
+# Launch (Usage & Customization)
+
+The Launch Configurable Keyboard can be customized and personalized in a variety of ways. It is recommended to remove unplug all USB cables and devices from your Launch before changing keycaps or switches.
+
+- [Connecting and Using the Launch](#connecting-and-using-launch)
```

Comment:
The section title does not have `the`, they should probably match unless there's a specific reason for them not to.

(The only place I've done this in the past is `Replacing the WiFi/Bluetooth module` linking to `Replacing the wireless card`, but the reason for that was so if someone's looking for WiFI/Bluetooth, they'll see it easier.)